fv3gfs-physics/fv3gfs/physics/stencils/microphysics.py

```diff
@@ -2234,8 +2226,31 @@ def setupm(self):
         self._csmlt = csmlt
         self._cgmlt = cgmlt
         self._ces0 = constants.EPS * es0
+        self._set_timestep(-1.0)
```

Comment:
is it -1 by default so we always take mp_time?

Reply:
if that's the case, maybe we should change it to have:
`self._mpdt = self.namelist.mp_time`?

Reply:
It's -1 by default so that it gets updated on the first timestep, and so that if we have a bug where we don't update the first time this is called then the behavior is weird enough that we should notice.